patreon/api.py

```diff
@@ -19,19 +20,32 @@ def fetch_campaign_and_patrons(self):
     def fetch_campaign(self):
         return self.__get_json('current_user/campaigns?include=rewards,creator,goals')
 
-    def fetch_page_of_pledges(self, campaign_id, page_size, cursor=None):
+    def fetch_page_of_pledges(self, campaign_id, page_size, cursor=None, return_cursor=False):
         url = 'campaigns/{0}/pledges?page%5Bcount%5D={1}'.format(campaign_id, str(page_size))
+
         if cursor:
             try:
                 cursor = self.__as_utc(cursor).isoformat()
             except AttributeError:
                 pass
             url += '&page%5Bcursor%5D={0}'.format(urllib.parse.quote(cursor))
```

Comment:
we should be consistent about module importation of `urllib` -- either this should be imported as a submodule up on line 3, or `parse_qs` and `urlparse` on line 3 should instead be fully specified on lines 38 & 39